packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -306,8 +306,12 @@ class ExpansionPanelList extends StatefulWidget {
 
   /// The callback that gets called whenever one of the expand/collapse buttons
   /// is pressed. The arguments passed to the callback are the index of the
-  /// to-be-expanded panel in the list and whether the panel is currently
-  /// expanded or not.
+  /// pressed panel and whether the panel is currently expanded or not.
+  ///
+  /// If ExpansionPanelList.radio is used, the callback may be called a
+  /// second time if a different panel was previously open. The arguments
+  /// passed to the second callback are the index of the closed panel and
```

Comment:
Index of the panel that will be closed or has been closed? Does the callback need to setState when it's notified about the closed/closing panel?

Reply:
I'll update the doc to clarify these points, but:
1) index of the panel that will be closed
2) callback does not need to call setState for radio mode, but does for regular ExpansionPanelList, so I'll make a note of that as well!